apps/perf-test/src/app/App.tsx

```diff
@@ -69,6 +68,7 @@ export const App = () => {
           ))}
         </div>
       )}
+      {/* eslint-enable react/forbid-component-props, react/jsx-no-bind */}
```

Comment:
Seems weird that we are enabling `forbid-component-props` at the end of the file when it is disabled globally.

Reply:
This is re-enabling the rules after the area where they needed to be disabled is finished.

Reply:
Oh I see what you mean in this case. What I said was probably what was originally done with the tslint equivalents?

Reply:
Probably.